app/models/agents/rss_agent.rb

```diff
@@ -105,6 +106,10 @@ def validate_options
         errors.add(:base, "Please provide 'expected_update_period_in_days' to indicate how many days can pass without an update before this Agent is considered to not be working")
       end
 
+      if options['max_ids'].present? && options['max_ids'].to_i < 1
```

Comment:
But 'url' and 'expected_update_period_in_days' use `options` as well for validation, yet in the rest of the agent `interpolated` is used. 
I guess I just don't quite understand the difference between the two and when which should be used. I could only figure out that `interpolated` appears to have its origins in `liquid_interpolatable`. Is it maybe the case that the `options` hash can have values with liquid formatting and the `interpolated` hash has the real values after the liquid formatting has been interpreted?